src/app/app-routing.module.ts

```diff
@@ -33,42 +35,63 @@ export const routes: Routes = [
   // Profile
   {
     path: ':entity',
+    resolve: {
+      context: ContextResolver
+    },
     loadChildren: './profile/profile.module#ProfileModule'
   },
 
   // Settings
   {
     path: ':entity/_settings',
+    resolve: {
+      context: ContextResolver
+    },
     loadChildren: './settings/settings.module#SettingsModule'
   },
 
   // Analyze
   {
     path: ':entity/:space',
+    resolve: {
+      context: ContextResolver
+    },
     loadChildren: './analyze/analyze.module#AnalyzeModule'
   },
 
   // Plan
   {
     path: ':entity/:space/plan',
+    resolve: {
+      context: ContextResolver
```

Comment:
What happens if you link directly to a submenu like the Planner Board menu. Does this resovler fire?